EIPS/eip-7876.md

```diff
@@ -0,0 +1,630 @@
+---
+eip: 7876
+title: Ethereum Network Configuration for DApps
+description: EVM-compatible network configuration suitable for every DApp supporting any custom network features through extensions.
+author: Bogdan Gusiev (@bogdan), Sergey Bomko (@aquiladev)
+discussions-to: https://ethereum-magicians.org/t/eip-xxxx-unified-network-configuration/22763
+status: Draft
+type: Meta
```

Comment:
meta eips are meant for eip process changes/updates, you may want to use informational category for it

Reply:
This category is not mentioned in [template](https://github.com/ethereum/EIPs/blob/master/eip-template.md). Maybe worth fixing.

```suggestion
type: Informational
```